src/resources/CallLog.js

```diff
@@ -0,0 +1,48 @@
+import Resource from './Resource';
+
+/**
+ * CallLog resource
+ */
+class CallLog extends Resource {
```

Comment:
This should be either `VoipCallRecord` (or perhaps `CallRecord`) to be in line with the models used in Track API